src/openrct2-ui/input/InputManager.cpp

```diff
@@ -209,6 +209,7 @@ void InputManager::Process(const InputEvent& e)
 
         if (e.DeviceKind == InputDeviceKind::Keyboard)
         {
+            // TODO: replace with event
```

Comment:
In OpenLoco, we've added an `OnKeyDown` event. I looked into adding one in OpenRCT2, however, I think we really need to sort out our window code before doing so...